### ISSUE TITLE
Prot warrior Shield Block

### ISSUE THREAD

**Issue by Leondezwart** [Valhalla-Gaming/Tracker#5109]
helloo Bigfighter here i recently came across shield block that it sometimes acts weird idk if that is normal cus im not used to it i just want to report it if in case its bugged ^^
The same thing also happens with shield barrier

http://www.wowhead.com/spell=112048/shield-barrier
http://www.wowhead.com/spell=2565/shield-block

**Comment by neano300** [comment]
yeah, i think in report this but not happends everytime so i can't make a good report about this, happends with all warrior specs
